res/model.xml

```diff
@@ -4958,7 +4958,7 @@
 				<show part="tail/demon$taur"/>
 			</case>
 			<case values="Tail.SPIDER_ADBOMEN">
-				<if test="lowerBody != LowerBody.ATLACH_NACHA">
+				<if test="lowerBody != LowerBody.DRIDER && lowerBody != LowerBody.ATLACH_NACHA">
```

Comment:
I've tested it with
```xml
<if test="not lowerBody isin [LowerBody.DRIDER, LowerBody.ATLACH_NACHA]">
```
and
```xml
<if test="lowerBody notin [LowerBody.DRIDER, LowerBody.ATLACH_NACHA]">
```
and while both worked as intended in Spred v2.3.0 it caused the abdomen to be always hidden in CoCX.